src/geointerface.jl

```diff
@@ -353,4 +357,7 @@ let pointtypes = (wkbPoint, wkbPoint25D, wkbPointM, wkbPointZM),
     function GeoInterface.geomtrait(geom::AbstractGeometry{wkbTriangle})
         return GeoInterface.TriangleTrait()
     end
+
+    geointerface_geomtype(trait::GeometryTraits) = IGeometry
+
```

Comment:
```suggestion
```
Whoops, didn't notice that this was already implemented in the same way at the top of the file.  Can be deleted.

Reply:
Yeah, it already works ;)